test/e2e/network/networking.go

```diff
@@ -646,28 +646,35 @@ var _ = common.SIGDescribe("SCTP [LinuxOnly]", func() {
 	f.NamespacePodSecurityEnforceLevel = admissionapi.LevelPrivileged
 
 	var cs clientset.Interface
+	var nodes *v1.NodeList
+	var sctpLoadedAtStart, sctpLoadedAtEnd bool
 
 	ginkgo.BeforeEach(func() {
 		cs = f.ClientSet
+
+		ginkgo.By("getting the state of the sctp module on nodes")
+		var err error
+		nodes, err = e2enode.GetBoundedReadySchedulableNodes(cs, 2)
+		framework.ExpectNoError(err)
+		sctpLoadedAtStart = CheckSCTPModuleLoadedOnNodes(f, nodes)
+	})
+
+	ginkgo.AfterEach(func() {
```

Comment:
you may prefer to use `DeferCleanup` for this?

https://onsi.github.io/ginkgo/#cleaning-up-our-cleanup-code-defercleanup

Reply:
I ended up removing the BeforeEach-refactoring commit; if we're not moving the NetworkPolicy test to be with the other two, then it makes less sense to keep them together as well; so I just left the SCTP service test in service.go and moved the SCTP hostport tests to networking.go (and left the SCTP NetworkPolicy test in netpol), and now there's no shared code to refactor out.